mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,274 @@
+    <string name="content_description_close_button">Close</string>
+    <!-- About content. Sentences are separated by two new lines -->
+    <string name="about_content">
+        %1$s puts you in control.
+        \n\n%1$s is produced by Mozilla. Our mission is to foster a healthy, open Internet.
+    </string>
+
+    <!-- Sessions -->
+    <!-- Title for the list of tabs in the current session -->
+    <string name="tabs_header_title">Current session</string>
+    <!-- Title for the list of tabs in the current private session -->
+    <string name="tabs_header_private_title">Private session</string>
+    <!-- Content description (not visible, for screen readers etc.): Add tab button. Adds a news tab when pressed -->
+    <string name="add_tab">Add Tab</string>
+    <!-- Content description (not visible, for screen readers etc.): Close tab button. Closes the current session when pressed -->
+    <string name="close_tab">Close tab</string>
+    <!-- Content description (not visible, for screen readers etc.): Opens the tab menu when pressed -->
+    <string name="tab_menu">Tab menu</string>
+    <!-- Content description (not visible, for screen readers etc.): Close tab button. Favicon for the session's site -->
+    <string name="favicon_content_description">Bookmark</string>
+    <!-- Button in the current session menu. Deletes the session when pressed -->
+    <string name="current_session_delete">Delete</string>
+    <!-- Button in the current session menu. Saves the session when pressed -->
+    <string name="current_session_save">Save</string>
+    <!-- Button in the current session menu. Opens the share menu when pressed -->
+    <string name="current_session_share">Share</string>
+    <!-- Content description (not visible, for screen readers etc.): Title icon for current session menu -->
+    <string name="current_session_image">Current session image</string>
+    <!-- Text for the button to save a session -->
+    <string name="session_save">Save session</string>
+    <!-- Text for the button to delete a session -->
+    <string name="session_delete">Delete session</string>
+    <!-- Text for the button to delete a single session -->
+    <string name="session_item_delete">Delete</string>
+    <!-- Text to tell the user how many more tabs this session has.
+        The first parameter is how many extra tabs the session has. -->
+    <string name="session_items_more">%1$d sites…</string>
+
+    <!-- History -->
+    <!-- Text for the button to clear all history -->
+    <string name="history_delete_all">Delete history</string>
+    <!-- Text for the button to delete a single history item -->
+    <string name="history_delete_item">Delete</string>
+    <!-- Text for the button to clear selected history items. The first parameter
+        is the number of items you have selected -->
+    <string name="history_delete_some">Delete %1$d items</string>
```

Comment:
Shouldn't this be something more like "Items to delete: %1$d" (or something along those lines, can't really think right now) for the plural?
CC @Pike

Reply:
I think that localizers can do that for themselves, as long as we don't support actual plurals.

For English, we should use the most natural expression, I think.

Reply:
@Pike do you mean we should use the local version of "items" even if it is only 1 item?  or that we should have a rule that if item count equals 0 or >1 use plural and if item count equals 1 use singular?

Reply:
Don't assume that 0 is plural, I suggest to special case 0 and 1, and use a string with parameter for the rest. That's as good as it gets right now.

Reply:
It would be definitely good to call out the limitation in our infrastructure, and to provide an alternative solution in the comment (that's what we do in Fennec, for example)